cassandra/src/test/java/zipkin/storage/cassandra/CassandraDependenciesTest.java

```diff
@@ -27,37 +29,39 @@
 import static zipkin.internal.Util.midnightUTC;
 
 public class CassandraDependenciesTest extends DependenciesTest {
-  private final CassandraStorage storage;
-
-  public CassandraDependenciesTest() {
-    this.storage = CassandraTestGraph.INSTANCE.storage.get();
-  }
+  @ClassRule public static LazyCassandraStorage storage =
+      new LazyCassandraStorage("openzipkin/zipkin-cassandra:1.29.2", "test_zipkin_dependency");
 
   @Override protected CassandraStorage storage() {
-    return storage;
+    return storage.get();
   }
 
   @Override public void clear() {
-    storage.clear();
+    storage().clear();
   }
 
   /**
    * This processes the job as if it were a batch. For each day we had traces, run the job again.
    */
-  @Override
-  public void processDependencies(List<Span> spans) {
-    CallbackCaptor<Void> callback = new CallbackCaptor<>();
-    storage.asyncSpanConsumer().accept(spans, callback);
-    callback.get();
+  @Override public void processDependencies(List<Span> spans) {
+    // TODO: this avoids overrunning the cluster, though we ought to deal with this upstream
```

Comment:
@openzipkin/cassandra note: one of our tests dumps 1000 spans and this dies on PoolBusyException. I'm avoiding this for now, but maybe we should do something fancy in the storage driver (or something un-fancy like repeating this there)